ImperatorToCK3/Source/Imperator/Provinces/ProvinceFactory.cpp

```diff
@@ -0,0 +1,54 @@
+﻿#include <numeric>
+#include "Log.h"
+#include "ParserHelpers.h"
+#include "CommonRegexes.h"
+#include "ProvinceFactory.h"
+#include "ProvinceName.h"
+
+
+Imperator::Province::Factory::Factory()
+{
+	registerKeyword("province_name", [this](std::istream& theStream) {
+		province->name = ProvinceName{ theStream }.getName();
+	});
+	registerSetter("culture", province->culture);
```

Comment:
As in the other review, setters don't work in factories.

Reply:
Replaced with registerKeyword ✔️